ThunderCloud/CollectionItemViewCell.swift

```diff
@@ -10,144 +10,210 @@ import UIKit
 import ThunderBasics
 import ThunderTable
 
-/// A UICollectionViewCell for use in a `CollectionListItem`
-open class CollectionItemViewCell: UICollectionViewCell {
+// MARK: - CollectionCellDisplayableStyle
+
+/// Previously an item being `enabled` would add a bold font and rounded rect to the label.
+/// However for some designs: when there is an `expiryDate`, the title label should not have the `enabled`style.
+public enum CollectionCellDisplayableStyle {
     
-    @IBOutlet weak var pinImageToBottomConstraint: NSLayoutConstraint!
+    /// Regular font, clear background, darkGray textColor
+    case `default`
     
-    @IBOutlet weak var titleImageSpacingConstraint: NSLayoutConstraint!
+    /// Bold font, main background, white textColor
+    case boldMain
     
-    private static let widthCalculationLabel = UILabel(frame: .zero)
+    /// Bold font, darkGray background, white textColor
+    case boldGray
     
-    /// The padding between the label/image view and the edges of the cell
-    static let cellPadding = UIEdgeInsets(top: 10, left: 8, bottom: 12, right: 8)
+    // MARK: Style
     
-    /// The padding between the title label and the image view
-    static let labelPadding = UIEdgeInsets(top: 2, left: 12, bottom: 2, right: 12)
+    fileprivate var weight: UIFont.Weight {
+        switch self {
+        case .default: return .regular
+        case .boldMain: return .bold
+        case .boldGray: return .bold
+        }
+    }
     
-    /// The spacing between the title label and the image view
-    static let labelImageSpacing: CGFloat = 6.0
+    fileprivate var backgroundColor: UIColor {
+        switch self {
+        case .default: return .clear
+        case .boldMain: return ThemeManager.shared.theme.mainColor
+        case .boldGray: return ThemeManager.shared.theme.darkGrayColor
+        }
+    }
     
-    /// The size of the image in the cell
-    static let imageSize = CGSize(width: 76, height: 76)
+    fileprivate var textColor: UIColor {
+        switch self {
+        case .default: return ThemeManager.shared.theme.darkGrayColor
+        case .boldMain: return ThemeManager.shared.theme.whiteColor
+        case .boldGray: return ThemeManager.shared.theme.whiteColor
+        }
+    }
+}
 
+// MARK: - CollectionItemViewCell
+
+/// A UICollectionViewCell for use in a `CollectionListItem`
+open class CollectionItemViewCell: UICollectionViewCell {
+    
+    /// Show circlular progress for items which do not degrade
+    public static var showProgressForNonExpirableItems = false
+    
+    /// Fix a  `CollectionCellDisplayableStyle` for the `titleLabel`. This for different designs across apps.
+    /// When nil, is `.boldMain` if `enabled`, otherwise `.default`
+    public static var fixedTitleLabelStyle: CollectionCellDisplayableStyle? = nil
```

Comment:
First time I saw you mutating a `static var` my mind was blown... for some reason I hadn't even realised that `static` could be a `var` you see `static let` all over the place for constants I just assumed it would be a compiler error! Such a nice way to have flags like this!

Reply:
Yeah true, a more common use case :P.  Tbh now there's 2 tempted to group it into a "configuration" struct